### DISCUSSION TITLE
netbox group & LDAP groups

### DISCUSSION THREAD

**Discussion by PenguinIsEvil** [netbox-community/netbox#16244] [Help Wanted!]
Hi all!
I have netbox version 4.0.2. 
Previously, is_staff users could view the netbox. after the update they can't. 
Question one, is this how it should be or did I not configure something when I arrived? 
Question two, is it possible to configure a specific LDAP group to assign a group created in the netbox? 
Adding a parameter to the config:
EXEMPT_VIEW_PERMISSIONS = [ '*' ]
It doesn’t suit me because with it the entire LDAP can see the entire netbox

**Comment by mdean-zen** [comment]
1) What version did you update from?
2) We utilize the auth ldap require group field, ours looks like this: 
`AUTH_LDAP_REQUIRE_GROUP = "cn=NETBOX-USERS,ou=Security Groups,dc=domain,dc=com"`
So only users in the NETBOX-USERS group can login, and then the permissions are defined in the AUTH_LDAP_USER_FLAGS such as the is_staff.
`AUTH_LDAP_USER_FLAGS_BY_GROUP = {
    "is_active": "cn=NETBOX-USERS,ou=Security Groups,dc=domain,dc=com",
    "is_staff": "cn=IT-Group,ou=Security Groups,dc=domain,dc=com",
    "is_superuser": "cn=IT-Group,ou=Security Groups,dc=domain,dc=com"
}
`
